03-Bash/kernel_git/kernel_stats.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# $1 - linux kernel sources path
+
+if [ $# -lt 1 ]
+then
+    echo "set kernel path as parametr"
+    exit 1
+fi
+
+# count *.c files
+declare -i C_COUNT
+declare -i CPP_COUNT
+declare -i PY_COUNT
+C_COUNT=0
+CPP_COUNT=0
+PY_COUNT=0
+
+for c_file in $(find "$1" -name "*.c" -type f)
```

Comment:
Consider using  wc -l for calculate the quantity of lines.

Reply:
Done